src/DetailsView/components/details-view-command-bar.tsx

```diff
@@ -60,41 +71,24 @@ export class DetailsViewCommandBar extends React.Component<DetailsViewCommandBar
         );
     }
 
-    private updatePersistedDescription = (value: string) => {
-        this.props.actionMessageCreator.addResultDescription(value);
-    };
-
-    private getExportDescription = () => {
-        return this.props.assessmentStoreData.resultDescription;
-    };
-
     private renderCommandButtons(): JSX.Element {
-        if (!this.props.renderExportAndStartOver) {
+        if (!this.props.reportExportComponentProps || !this.props.renderStartOver) {
             return null;
         }
-        const { deps, assessmentStoreData, assessmentsProvider, featureFlagStoreData, tabStoreData } = this.props;
-        const reportGenerator = deps.reportGenerator;
         const selectedTest = this.props.assessmentStoreData.assessmentNavState.selectedTestType;
         const test = this.props.assessmentsProvider.forType(selectedTest);
-        const htmlGenerator = reportGenerator.generateAssessmentReport.bind(
-            reportGenerator,
-            assessmentStoreData,
-            assessmentsProvider,
-            featureFlagStoreData,
-            tabStoreData,
-        );
 
         return (
             <div className="details-view-command-buttons">
                 <ReportExportComponent
-                    deps={deps}
-                    reportGenerator={reportGenerator}
-                    pageTitle={tabStoreData.title}
-                    exportResultsType={'Assessment'}
-                    scanDate={deps.getCurrentDate()}
-                    htmlGenerator={htmlGenerator}
-                    updatePersistedDescription={this.updatePersistedDescription}
-                    getExportDescription={this.getExportDescription}
+                    deps={this.props.reportExportComponentProps.deps}
```

Comment:
Might be a good idea to use the spread operator instead:

```
<ReportExportComponent {...this.props.reportExportComponentProps} />
```

Reply:
Done